media-art-background.js

```diff
@@ -42,6 +42,8 @@ function setBackground(root, appLayout, lovelace, bgroundElem) {
     let entityName = entity.entity || entity;
     let entityValidStates = entity.valid_states || ['playing'];
     let entityImageAttribute = entity.image_attribute || 'entity_picture';
+    let entityValidViews = entity.views; //get a list of valid views
+    let currentview = window.location.pathname.split('/')[2]; // determine current view
```

Comment:
While the card seems to be broken on all other views currently, I don't think we should try to limit ourselves. If we can do cover art background on more screens at a certain point, it would be good to have. So I think we want to just deal with the whole pathname here.

Reply:
Apologies for the delay.

Im not sure I understand.

The way I have it is that if views are specified in the entities item the background is only changed when a matching view is displayed.

If the views are not specified it changed every lovelace view.

![image](https://user-images.githubusercontent.com/29498865/114679103-c7f33680-9d03-11eb-87a1-2e3bba31be62.png)

![image](https://user-images.githubusercontent.com/29498865/114679252-e822f580-9d03-11eb-8930-3d69a6552a57.png)
views disabled - now changes every screen

Reply:
I mean not removing /lovelace from it, so in the future it may also be possible to use it on /calendar or so.

Reply:
I thought that these plugins only are used in lovelace dashboards. so the url will always be 
https://homeassistant.ip/lovelace-dashboard-name/dashboard-view

Reply:
Well, Home Assistant does load all these plugins on other pages like /calendar, /map, /logbook and so too. So even though the plugin currently errors on those, I'd rather not make it harder in the future to support this.

Reply:
Ah I see what you mean, yeah that makes perfect sense.

Reply:
ok I have modified it so it should not error on non-lovelace pages.